doc/source/whatsnew/v1.1.0.rst

```diff
@@ -220,19 +220,7 @@ Other enhancements
   This can be used to set a custom compression level, e.g.,
   ``df.to_csv(path, compression={'method': 'gzip', 'compresslevel': 1}``
   (:issue:`33196`)
-- :meth:`Series.update` now accepts objects that can be coerced to a :class:`Series`,
-  such as ``dict`` and ``list``, mirroring the behavior of :meth:`DataFrame.update` (:issue:`33215`)
-- :meth:`~pandas.core.groupby.GroupBy.transform` and :meth:`~pandas.core.groupby.GroupBy.aggregate` has gained ``engine`` and ``engine_kwargs`` arguments that supports executing functions with ``Numba`` (:issue:`32854`, :issue:`33388`)
-- :meth:`~pandas.core.resample.Resampler.interpolate` now supports SciPy interpolation method :class:`scipy.interpolate.CubicSpline` as method ``cubicspline`` (:issue:`33670`)
-- The ``ExtensionArray`` class has now an :meth:`~pandas.arrays.ExtensionArray.equals`
-  method, similarly to :meth:`Series.equals` (:issue:`27081`).
-- The minimum suppported dta version has increased to 105 in :meth:`~pandas.io.stata.read_stata` and :class:`~pandas.io.stata.StataReader`  (:issue:`26667`).
-- :meth:`~pandas.core.frame.DataFrame.to_stata` supports compression using the ``compression``
-  keyword argument. Compression can either be inferred or explicitly set using a string or a
-  dictionary containing both the method and any additional arguments that are passed to the
-  compression library. Compression was also added to the low-level Stata-file writers
-  :class:`~pandas.io.stata.StataWriter`, :class:`~pandas.io.stata.StataWriter117`,
-  and :class:`~pandas.io.stata.StataWriterUTF8` (:issue:`26599`).
+- :meth:`HDFStore.put` now accepts `track_times` parameter. Parameter is passed to ``create_table`` method of ``PyTables`` (:issue:`32682`).
```

Comment:
don't change anything else, pls revert, then add this parameter after

Reply:
o-o, first time I tried to solve conflict via github page and failed :-) ...
rebased and fixed